docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# Issue Template
+
+* Generally speaking, issues should consist of bugs in SLEAP that will require a pull request. For general questions and suggestions for enhancements, you can start a new discussion [here](https://github.com/talmolab/sleap/discussions)
```

Comment:
Lets link the general question to https://github.com/talmolab/sleap/discussions/categories/help, and suggestions to https://github.com/talmolab/sleap/discussions/categories/ideas